e2e/tests/examples/data-driven.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { config } from 'resources/workbench-config';
+import GoogleLoginPage from 'app/page/google-login';
+import { withPage } from 'libs/page-manager';
+
+describe('Login tests:', () => {
+  const urls = [
+    { value: `${config.uiBaseUrl}${config.workspacesUrlPath}` },
+    { value: config.uiBaseUrl },
+    { value: `${config.uiBaseUrl}${config.profileUrlPath}` }
+  ];
+
+  test.each(urls)('Redirects to login page', async (url) => {
```

Comment:
nit: `async ({value}) => {`

Reply:
👍🏻 I updated `login.spec.ts`: replacing old `redirects to login page` test with this new data-driven test.